src/ptc/util/jms.clj

```diff
@@ -57,12 +57,16 @@
    ::param  true     :SAMPLE_ID                           :sampleId
    ::param  true     :SAMPLE_LSID                         :sampleLsid])
 
+(def push-or-copy-keys
+  "WFL request notification keys for files that may already exist in the cloud."
+  [:green_idat_cloud_path :red_idat_cloud_path])
+
 (def required-jms-keys
   "Sort all the keys required to handle a JMS message."
```

Comment:
No longer sorts?

Reply:
Good catch! I updated the docstring and sorted the missing keys in the error message